### PR TITLE
Update to RAWcooked man page

### PR DESCRIPTION
**Pull request by digitensions**
Some corrections based on latest --help page, and a few additions such as --accept-gaps and --no-accept-gaps, additional descriptions for --check-padding flags and some potential examples that could be useful? Just putting a few ideas down for feedback, can add more/amend if this is thought to be a worthwhile idea.  Apologies for errors, it's been a while since I made a PR!

### CODE REVIEW

**Comment by bturkus** [comment]
this is awesome!

File: Source/CLI/rawcooked.1 [hunk 1]
```diff
@@ -207,6 +202,13 @@ Default value is \fI${Input}.framemd5\fR.
 Do not compute of framemd5 of input frames. (see above)."
 .br
 Is default.
+.TP
+.B --accept-gaps
+Use if there are missing files within the sequence numbering. RAWcooked creates a concatenated list of all files ensuring the sequence can be encoded.
+.TP
+.B --no-accept-gaps
+Do not accept-gaps within the sequence numbering. FFmpeg will fail any encoding attempts where gaps are present.
+.TP
```
Comment (retokromer) [inline Source/CLI/rawcooked.1:211]: @JeromeMartinez Should one of `--accept-gaps` and `--no-accept-gaps` not be default?

**Review by retokromer — APPROVED** [review]
One question, otherwise LGTM.

File: Source/CLI/rawcooked.1 (hunk 1 above)
Comment (digitensions) [reply Source/CLI/rawcooked.1:211]: Ah thanks @retokromer I missed that!

Comment (JeromeMartinez) [reply Source/CLI/rawcooked.1:213]: @retokromer I added the default value (`--no-accept-gaps`).
@digitensions I combined in one commit, resolved the merge conflict, added the default value, and forced-pushed the commit to your branch and this PR. Let me know if something is missing or has an issue.

**Pull request merged**